README.md

```diff
@@ -29,11 +37,11 @@ This will trigger the action when an issue or pull request is opened. You'll nee
 By default, the label `sponsor` will be applied. If you have a different label you want to use, you can set that:
 
 ```yaml
-- uses: JasonEtco/is-sponsor-label-action@master
-  with:
-    label: Sponsor Request ❤️
-  env:
-    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - uses: JasonEtco/is-sponsor-label-action@master
```

Comment:
Were these indentation changes intentional?

Reply:
I was aligning it to match the section above, since this is referring to a nested part of the yml file rather than the full file. We could un-indent though, I realize it's kinda weird.

Reply:
Ah ok - I don't prefer either way, so I'm fine with it as-is, just making sure it was intentional 👍